packages/core/src/shared/utilities/processUtils.ts

```diff
@@ -100,6 +100,13 @@ export class ChildProcess {
         // TODO: allow caller to use the various loggers instead of just the single one
         this.#log = baseOptions.logging !== 'no' ? logger.getLogger() : logger.getNullLogger()
     }
+    public static async exec(
```

Comment:
can we call this `run`? should avoid different names for the same thing.

Reply:
Original intention was to mirror `child_process` `exec`, but I agree the consistency is more important.